exonum-light-client/CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+## 0.3.0 - 2019-07-22
+
+The third release of Exonum Java Light Client which improves
+a convenience in working with blocks and brings support of
+the prefix in URLs.
```

Comment:
I'd clarify it a little: 

… and brings support of/allows specifying _a custom path prefix to Exonum API_.